Plugin.php

```diff
@@ -79,7 +80,9 @@ protected function extendBackendControllers(): void
             // @TODO: Handle cache busting through some other method
             $cssLastModified = filemtime(plugins_path('winter/tailwindui/assets/css/dist/backend.css'));
 
-            $controller->addCss(Url::asset('/plugins/winter/tailwindui/assets/css/dist/backend.css'), (string) $cssLastModified);
+            if (App::runningInBackend()) {
+                $controller->addCss(Url::asset('/plugins/winter/tailwindui/assets/css/dist/backend.css'), (string) $cssLastModified);
+            }
```

Comment:
This is extending the BaseBackendController, not the frontend controller. Is this just to prevent EasyForms from loading this style in the frontend?

Reply:
Yes, but what's the point of adding this css file if we're in frontend context?

Reply:
This shouldn't run in a frontend context when it's in a backend controller. The fact that it does is a somewhat hacky aspect of how the EasyForms plugin works; and it should be up to the EasyForms plugin to control for that.

Reply:
Fair enough, I'll let you handle this in your plugin then.